scripts/windows_install_mingw64.ps1

```diff
@@ -0,0 +1,33 @@
+$ErrorActionPreference = "Stop"
+
+echo "starting mingw64 installation"
+
+Invoke-WebRequest -UseBasicParsing -uri "https://github.com/msys2/msys2-installer/releases/download/nightly-x86_64/msys2-base-x86_64-latest.sfx.exe" -OutFile msys2.exe ; 
```

Comment:
I  found Invoke-WebRequest be quite slow, turning to curl.exe (note not 'curl', which is annoyingly is a powershell alias)

Reply:
Yup, takes quite long as done above, but is quite fast with curl (tested the latter locally).

Reply:
changed it with curl.exe